generator/postgres/postgres_generator.go

```diff
@@ -60,7 +60,18 @@ func GenerateDSN(dsn, schema, destDir string, templates ...template.Template) (e
 		generatorTemplate = templates[0]
 	}
 
-	schemaMetadata := metadata.GetSchema(db, &postgresQuerySet{}, schema)
+	var isRedshift bool
+	err = db.QueryRow("SELECT version() LIKE '%Redshift%';").Scan(&isRedshift)
+	if err != nil {
```

Comment:
No need for if.

Reply:
What do you suggest to do to decide which query set to use then?

Reply:
Updated.